### PR TITLE
Add stale action

### PR DESCRIPTION
**Pull request by skjnldsv**
- [x] We need https://github.com/actions/stale/issues/17

Replacing https://github.com/nextcloud/server/blob/master/.github/stale.yml

:warning: set in dry-mode just to see

### CODE REVIEW

**Comment by skjnldsv** [comment]
@PVince81 can we consider that again for server? :innocent:

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          This issue has been automatically marked as stale because it has not had
+          recent activity and seems to be missing some essential information.
+          It will be closed if no further activity occurs. Thank you
+          for your contributions.
+        stale-issue-label: 'stale'
+        only-labels: 'needs info'
+        exempt-issue-labels: '1. to develop,2. developing,3. to review,4. to release,security'
+        days-before-stale: 30
+        days-before-close: 14
+        debug-only: true
```
Comment (szaimen) [inline .github/workflows/stale.yml:30]: ?

**Review by szaimen — APPROVED** [review]

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    runs-on: ubuntu-latest
+    
+    permissions:
+      issues: write
+    
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.COMMAND_BOT_PAT }}
+        stale-issue-message: >
```
Comment (szaimen) [inline .github/workflows/stale.yml:20]:
```suggestion
        stale-issue-message: |
```
?

**Review by szaimen — APPROVED** [review]

File: .github/workflows/stale.yml (hunk 1 above)
Comment (skjnldsv) [reply .github/workflows/stale.yml:30]: > :warning: set in dry-mode just to see

File: .github/workflows/stale.yml (hunk 2 above)
Comment (skjnldsv) [reply .github/workflows/stale.yml:20]: [Allowed in yaml ](https://yaml.org/spec/1.2/spec.html#id2795688)

**Review by artonge — APPROVED** [review]

**Pull request merged**